tests/mentorship_relation/test_api_accept_request.py

```diff
@@ -52,7 +53,7 @@ def test_accept_mentorship_request(self, send_email_function):
                 headers=get_test_request_header(self.second_user.id),
             )
 
-            self.assertEqual(200, response.status_code)
+            self.assertEqual(HTTPStatus.OK, response.status_code)
```

Comment:
It should be like:-
```suggestion
            self.assertEqual(HTTPStatus.OK.value, response.status_code)
```

Reply:
it's ok, without that `.value` here :) This is only needed in the case of exposing the values on Swagger UI. For feature and test code this works, so should be good without the extra "get value". Thank you for being so thorough anyways @devkapilbansal !